saltychat/SaltyClient/External/Tasks.cs

```diff
@@ -0,0 +1,52 @@
+﻿using CitizenFX.Core;
+using CitizenFX.Core.Native;
+using System;
+using System.Threading.Tasks;
+
+namespace RedM.External
+{
+    public class Tasks
+    {
+        private Ped Ped { get; }
+
+        internal Tasks(Ped ped)
+        {
+            Ped = ped;
```

Comment:
I'm still a big fan of `this` :)
```suggestion
            this.Ped = ped;
```